Std/Data/List/Init/Lemmas.lean

```diff
@@ -4,6 +4,10 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Parikshit Khanna, Jeremy Avigad, Leonardo de Moura, Floris van Doorn, Mario Carneiro
 -/
 
+-- TODO: I'm not sure where this should live.
```

Comment:
Is `Mathlib/Init/Data/Fin/Basic.lean` a good place?

Reply:
@jcommelin, I think you missed this is a PR at Std. :-)

Reply:
Nevertheless the same idea but `Std/Data/Fin/Init.lean` is good.

Reply:
the naming convention is `Std.Data.Fin.Init.Basic`